packages/styles/scss/patterns/cardarray/_cardarray-item.scss

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../components/link-with-icon/link-with-icon';
+
+@mixin cardarray-item {
+  .#{$prefix}--cardarray-item {
+    padding-top: $carbon--layout-01;
+    padding-left: $carbon--layout-01;
+    border: solid 1px #ffffff;
+    background: $carbon--gray-10;
+    display: flex;
+    flex-direction: column;
+    flex: 100%;
+    @include carbon--breakpoint('lg') {
+      flex: 0 50%;
+    }
+    &__title {
+      margin-bottom: $carbon--layout-03;
+      width: 90%;
+      @include carbon--type-style('heading-02');
```

Comment:
Looking at the sketch file, for the title and the paragraph content the max-width is 640px;